ts/Core/Pointer.ts

```diff
@@ -1830,29 +1855,46 @@ class Pointer {
      * @private
      * @function Highcharts.Pointer#scaleGroups
      */
-    public scaleGroups(attribs?: Series.PlotBoxObject, clip?: boolean): void {
+    public scaleGroups(attribs?: Series.PlotBoxObject, clip?: BBoxObject, pinchedAxes?: Axis[]): void {
 
         const chart = this.chart;
-
+        let axes = pinchedAxes || chart.axes;
         // Scale each series
-        chart.series.forEach(function (series): void {
-            const seriesAttribs = attribs || series.getPlotBox(); // #1701
-            if (series.xAxis && series.xAxis.zoomEnabled && series.group) {
-                series.group.attr(seriesAttribs);
-                if (series.markerGroup) {
-                    series.markerGroup.attr(seriesAttribs);
-                    series.markerGroup.clip(
-                        clip ? (chart.clipRect as any) : (null as any)
-                    );
+        axes.forEach((axis: Axis): void => {
+            axis.series.forEach((series: Series): void => {
+                const seriesAttribs = attribs || series.getPlotBox(), // #1701
+                    isX = axis.isXAxis,
+                    attr: Partial<Series.PlotBoxObject> = {};
+
+                if (isX) {
+                    attr.scaleX = seriesAttribs.scaleX;
+                    const transformScale = chart.inverted ? 1 / attr.scaleX : attr.scaleX;
+                    attr.translateX = seriesAttribs.translateX + (attribs ? (transformScale * axis.pos) : 0);
+                } else {
+                    attr.scaleY = seriesAttribs.scaleY;
+                    const transformScale = chart.inverted ? 1 / attr.scaleY : attr.scaleY;
+                    attr.translateY = seriesAttribs.translateY + (attribs ? (transformScale * axis.pos) : 0);
```

Comment:
Generating `attr` object could be moved to `axes.forEach()` loop, instead of calculating this in `series.forEach()` loop.